railties/lib/rails/generators.rb

```diff
@@ -156,12 +156,25 @@ def self.invoke(namespace, args=ARGV, config={})
         args << "--help" if args.empty? && klass.arguments.any?(&:required?)
         klass.start(args, config)
       else
-        options     = sorted_groups.map(&:last).flatten
-        suggestions = options.sort_by {|suggested| levenshtein_distance(namespace.to_s, suggested) }.first(3)
-        msg =  "Could not find generator '#{namespace}'. "
-        msg << "Maybe you meant #{ suggestions.map {|s| "'#{s}'"}.join(" or ") }\n"
-        msg << "Run `rails generate --help` for more options."
-        puts msg
+        unless sorted_groups.group_by(&:size).min.first > namespace.length
```

Comment:
Actually this line exists to reject too short words. `sorted_groups.group_by(&:size).min.first` returns the length of the shortest commands available, which is  `3` in the rails master.
Unless we have another algorithm degicated for short inputs to suggest only commands starting with those chars (e.g. <kbd>ap</kbd> -> `API`, `apple` or `application`), I don't see point of making a suggestion for just 1 or 2 letter inputs.